process_request/helpers.py

```diff
@@ -12,18 +12,16 @@ def get_collection_creator(archival_object):
     return ",".join(creators)
 
 
-def get_location(instance_json):
-    """Takes json for a single instance of an archival object that has the _resolved parameter on top_containers and returns the title field of the container location for each location."""
-    top_container_info = instance_json.get("sub_container").get("top_container").get("_resolved")
+def get_location(top_container_info):
+    """Takes the _resolved json for a top container (with the _resolved parameter also on container locations) from an archival object and returns the title field of the container location for each location. C"""
     locations = []
     for c in top_container_info.get("container_locations"):
         locations.append(c.get("_resolved").get("title"))
```

Comment:
this *could* be done in one line via list comprehension:
```suggestion
    locations = [c.get("_resolved").get("title") for c in top_container_info.get("container_locations")]
```